invenio_app_ils/document_requests/api.py

```diff
@@ -74,6 +74,16 @@ def validate_rejection(self, document_pid, state, reject_reason):
                 "without providing a document_pid."
             )
 
+    def validate_acceptance(self, document_pid, physical_item_provider, state):
+        """Validate rejection is correct."""
+        if state == "ACCEPTED" and (
```

Comment:
I think we should quickly chat maybe?
The endpoint is requiring both (doc and provider), see [here](https://github.com/inveniosoftware/invenio-app-ils/blob/0280d21e755b6f68c0e4f501a06f7539333bde64/invenio_app_ils/document_requests/views.py#L195) and it is not great that we split the validation.